missating/More-Recipes
# Ch client setup

## What does this PR do?
Configure development environment for client-side using webpack

## How to manually test this?
Run npm install to install all needed dependencies then run <code> npm run dev </code> to start up webpack and go tohttp://localhost:8080/ to check it out.
